recipe/meta.yaml

```diff
@@ -81,15 +81,15 @@ outputs:
         - itsdangerous
         - jinja2
         - minio
-        - pydantic <2.0a0
+        - pydantic >=2.0
         - pyjwt
         - python >={{ python_min }}
         - python-docker
         - python-multipart
         - pyyaml >=6.0.1
         - redis-py
         - requests
-        - sqlalchemy <2.0a0
```

Comment:
Is there a reason why this pin is different from what [we have in our repository `meta.yaml`](https://github.com/conda-incubator/conda-store/blob/f8875ca977fe8309b521e23a5582af7cfd64ec6b/recipe/meta.yaml#L91-L92)?
There we have `- sqlalchemy >=2.0,<2.1` same as in the `pyproject.toml`.

We should make sure that these three files (barring any exceptional circumstances).

Reply:
Good catch, updated!